tools/challenge-helper-scripts/create-project.ts

```diff
@@ -41,34 +47,136 @@ interface CreateProjectArgs {
   superBlock: SuperBlocks;
   block: string;
   helpCategory: string;
-  order: number;
+  blockType?: string;
+  blockLayout?: string;
+  questionCount?: number;
+  order?: number;
+  chapter?: string;
+  position?: number;
+  module?: string;
   title?: string;
 }
 
-async function createProject(
-  superBlock: SuperBlocks,
-  block: string,
-  helpCategory: string,
-  order: number,
-  title?: string
-) {
-  if (!title) {
-    title = block;
+async function createProject(projectArgs: CreateProjectArgs) {
+  if (!projectArgs.title) {
+    projectArgs.title = projectArgs.block;
   }
-  void updateIntroJson(superBlock, block, title);
 
-  const challengeId = await createFirstChallenge(superBlock, block);
-  void createMetaJson(block, title, helpCategory, challengeId);
+  const order = projectArgs.order!;
+  const chapter = projectArgs.chapter!;
+  const module = projectArgs.module!;
```

Comment:
```suggestion
  const order = projectArgs.order;
  const chapter = projectArgs.chapter;
  const module = projectArgs.module;
```
These aren't guaranteed to exist.